forum/forum/logging.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from typing import Literal
+
+
+class ColoredFormatter(logging.Formatter):
+    grey = "\x1b[38;20m"
+    green = "\u001b[32m"
+    yellow = "\x1b[33;20m"
+    red = "\x1b[31;20m"
+    magenta = "\u001b[35m"
+    reset = "\x1b[0m"
+
+    def __init__(
+        self,
+        fmt: str | None = None,
+        datefmt: str | None = None,
+        style: Literal['%'] | Literal['{'] | Literal['$'] = "%",
+        validate: bool = True,
+        *,
+        defaults=None
+    ) -> None:
+        super().__init__(fmt, datefmt, style, validate, defaults=defaults)
+        self.FORMATS = {
+            logging.DEBUG: self.grey + self._fmt + self.reset,
+            logging.INFO: self.green + self._fmt + self.reset,
+            logging.WARNING: self.yellow + self._fmt + self.reset,
+            logging.ERROR: self.red + self._fmt + self.reset,
+            logging.CRITICAL: self.magenta + self._fmt + self.reset
+        }
+
+    def format(self, record):
+        log_fmt = self.FORMATS.get(record.levelno)
+        formatter = logging.Formatter(log_fmt)
+        return formatter.format(record)
+
+
+logger = logging.getLogger("django")
```

Comment:
Add a handler to the logger and set the formatter.
Set the log level for the logger.

Reply:
This things are described in `settings.py` file LOGGING variable, so we can use such construction to get ther logger.